ql/src/java/org/apache/hadoop/hive/ql/security/authorization/plugin/metastore/HiveMetaStoreAuthorizableEvent.java

```diff
@@ -64,6 +64,12 @@ protected HivePrivilegeObject getHivePrivilegeObject(Table table) {
         table.getOwner(), table.getOwnerType());
   }
 
+  protected HivePrivilegeObject getHivePrivilegeObjectStorageHandlerUri(String storagehandler_uri, Table table) {
+    return new HivePrivilegeObject(HivePrivilegeObject.HivePrivilegeObjectType.STORAGEHANDLER_URI, null, storagehandler_uri,
+             null, null, HivePrivilegeObject.HivePrivObjectActionType.OTHER, null, null,
```

Comment:
seems we don't use this method, can we remove it then?

Reply:
Sure.